DependencyInjection/MonologExtension.php

```diff
@@ -59,6 +60,8 @@ public function load(array $configs, ContainerBuilder $container)
             // always autowire the main logger, require Symfony >= 2.8
             if (method_exists('Symfony\Component\DependencyInjection\Definition', 'addAutowiringType')) {
                 $container->getDefinition('monolog.logger')->addAutowiringType('Psr\Log\LoggerInterface');
+            } else {
```

Comment:
that's wrong. Symfony 3.3 does have the method, so it will never enter the ``else`` part, while it is the version where you want the alias.

currently, you only define the alias for Symfony < 2.8, where it is useless